src/locales/ko/dialogue.json

```
@@ -935,112 +935,112 @@
   },
   "stat_trainer_buck": {
     "encounter": {
-      "1": "...I'm telling you right now. I'm seriously tough. Act surprised!",
-      "2": "I can feel my Pokémon shivering inside their Pokéballs!"
+      "1": "...말해두겠지만 나 강하다고!\n놀라기나 해라!",
+      "2": "몬스터볼 안에서 내 포켓몬들이 떨고 있다!\n이게 흥분해서 떨린다는 것이구나!"
     },
     "victory": {
-      "1": "Heeheehee!\nSo hot, you!",
-      "2": "Heeheehee!\nSo hot, you!"
+      "1": "이히히!\n너 열정이 뜨거운걸!!",
+      "2": "이히히!\n너 열정이 뜨거운걸!!"
     },
     "defeat": {
-      "1": "Whoa! You're all out of gas, I guess.",
-      "2": "Whoa! You're all out of gas, I guess."
+      "1": "우왓!\n연료가 다 떨어졌나본데?",
+      "2": "우왓!\n연료가 다 떨어졌나본데?"
     }
   },
   "stat_trainer_cheryl": {
     "encounter": {
-      "1": "My Pokémon have been itching for a battle.",
-      "2": "I should warn you, my Pokémon can be quite rambunctious."
+      "1": "내 포켓몬들\n싸우고 싶어서 근질근질하고 있었어",
```

```suggestion
      "1": "내 포켓몬들\n싸우고 싶어서 근질근질하고 있었어.",
```
특별한 경우가 아니라면 대사 스크립트 끝에는 온점을 찍어두는 게 좋지 않을까 싶습니다. 지금 폰이라 한 번에 리뷰가 안 되네요,,

---

원문에서 문장부호 없는 경우가 있어서 빼뒀는데 부호는 알아서 넣어야겠네요